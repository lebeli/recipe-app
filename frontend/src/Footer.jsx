import React, { Component } from "react";
import "./Footer.scss";

class Footer extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return <div className="Footer"></div>;
  }
}

export default Footer;
