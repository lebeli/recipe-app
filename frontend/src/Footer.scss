@import "./styles/styles.scss";

.Footer {
  background-color: $primaryColor;
  position: absolute;
  bottom: 0;
  width: 100%;
  height: 100px;
}
