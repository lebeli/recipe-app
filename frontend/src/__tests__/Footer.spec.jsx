import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import Footer from "../Footer";

describe("<Footer />", () => {
  test("should render elements", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find(".Footer"));
  });

  test("renders correctly", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
