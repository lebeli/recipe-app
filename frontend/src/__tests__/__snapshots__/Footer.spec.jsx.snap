// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<div
  className="Footer"
>
  <div
    className="MuiBox-root MuiBox-root-2"
  >
    <p>
      <a
        href="#"
      >
        Impressum
      </a>
       | 
      <a
        href="#"
      >
        Datenschutz
      </a>
    </p>
  </div>
</div>
`;
